app/assets/javascripts/discourse/app/models/post-stream.js

```diff
@@ -993,7 +1000,15 @@ export default RestModel.extend({
     const data = { post_ids: postIds, include_suggested: includeSuggested };
     const store = this.store;
 
-    return ajax(url, { data }).then((result) => {
+    let headers = {};
+    if (opts && opts.background) {
```

Comment:
You can use this now I think, I'm still fixing few issues with babels and plugins but fore core that should be good:

```suggestion
    if (opts?.background) {
```